packages/d3fc-webgl/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-2016 Scott Logic Ltd.
```

Comment:
we should update the copyright to the current year : )

Reply:
This might be better off done separately on all packages to avoid having different licenses scattered throughout the repo.